### ISSUE TITLE
Region rendering slow for boundaries

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#896]
Reproduce:

- `MoBIE( "https://github.com/mobie/mouse-embryo-spatial-transcriptomics-project", MoBIESettings.settings().view( "stitched-view" )`
- Render `positions` as boundaries with a thickness of `2`

Zooming in and out is much smoother when _not_ rendering the regions as boundaries.

@constantinpape can you confirm this?

**Comment by constantinpape** [comment]
Yes, I have already noticed this.

**Comment by tischi** [comment]
How is it for 
- the HTM project?
- the platy segmentation?

(I can also test myself later)

**Comment by constantinpape** [comment]
Sorry, I haven't done any systematic tests on this either. I just noticed it recently for the stitched view here as well. Best if you check this later yourself.

**Comment by tischi** [comment]
- Platy-cells: no issue
- HTM: same issue
- Spatial-oimcs spots: no issue

This suggests that this is specific to `regions`.

**Comment by tischi** [comment]
In fact it also does not really work:

<img width="285" alt="image" src="https://user-images.githubusercontent.com/2157566/201089474-1b9aaea8-677f-42a1-9b2a-f6ab1d09c707.png">

Missing boundaries.

**Comment by tischi** [comment]
Only if one waits long enough for the highest resolution. I think we had an issue about this already.

**Comment by tischi** [comment]
Most time seems to be spend here:

<img width="698" alt="image" src="https://user-images.githubusercontent.com/2157566/201089921-5866b25a-53e4-4e97-af3e-b3675e75ee2f.png">

**Comment by tischi** [comment]
I added a FIXME into `TableSawAnnotatedRegion`:

```
	@Override
	public RealMaskRealInterval getMask()
	{
		// Update every time, because the position of the images
		// maybe have changed.
		// FIXME: we should cache this! This is too expensive!
		//   probably needs some listener mechanism!
		final Set< Image< ? > > regionImages = DataStore.getImageSet( imageNames );
		final RealMaskRealInterval unionMask = TransformHelper.getUnionMask( regionImages, timePoint() );
		return unionMask;
	}
```

**Comment by tischi** [comment]
should be fixed with `3.0.5`
